Adds pre-commit hooks for black, flake8, isort

chrismoradi opened the pull request:
**Description**

This PR fixes #194

Ran into this issue with new install of [black](https://github.com/psf/black/issues/2964), but the pre-commit hook should be correct.

Questions for reviewer:
- Is there somewhere I should add documentation on installing and running pre-commit hooks (e.g., README.md or CONTRIBUTING.md)?
- Do you use the default line length with black? Any other settings that should be specified for other hooks?

<!--
Thank you for contributing to Robyn! 

Contributing Conventions:

1. Include descriptive PR titles.
2. Build and test your changes before submitting a PR. 
-->

sansyrox commented:
Hi @chrismoradi ,

Thank you for the contribution 😄 

I have a few suggestions:
- Can you set the `default_stage`(https://pre-commit.com/#top_level-default_stages) to `push`?
- I use max line length of `160` in flake8 (https://github.com/sansyrox/robyn/blob/main/.flake8)
- We do have a CONTRIBUTORS.MD file but it is a little generic at the moment. Can you add the install instructions in README for now?
This section(https://github.com/sansyrox/robyn#%EF%B8%8F-to-develop-locally) should work.

chrismoradi commented:
@sansyrox, I added the `default_stage` and install instructions to the README (please let me know if you want me to rework that).

For flake8, it looks like it's using the config settings already in `.flake8`. I tested changing the max line length to 90 and it starts showing line length errors. Happy to add this to the pre-commit config but thought it would add extra maintenance to have this setting in 2 places.

Also added a config line for `black` to use 160 max line length.

sansyrox reviewed (approved):
LGTM! 🚀 :shipit:
